src/main/java/com/uc/budgettracker/BudgetTrackerApplication.java

```diff
@@ -1,15 +1,15 @@
-package com.uc.budgettracker;
-
-import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
-
-@EnableJpaRepositories("com.uc.budgettracker.repositories")
-@SpringBootApplication
-public class BudgettrackerApplication {
-
-	public static void main(String[] args) {
-		SpringApplication.run(BudgettrackerApplication.class, args);
-	}
-
-}
+package com.uc.budgettracker;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+
+@EnableJpaRepositories("com.uc.budgettracker.repositories")
+@SpringBootApplication
+public class BudgetTrackerApplication {

```

Comment:
UpperCamelCase is the proper naming convention for Java classes.  Budgettracker is not a word, so it should be styled as BudgetTracker.